build.sbt

```diff
@@ -285,6 +283,20 @@ lazy val semanticdbScalac = project
       }).transform(node).head
     }
   )
+  .dependsOn(semanticdbScalac)
+
+
+lazy val semanticdbScalac = project
```

Comment:
How about `semanticdbScalacLibrary`?

Reply:
semanticdbScalac is better IMO since it matches the directory name and I run `semanticdbScalac/test` from the console all the time (which is already too long IMO), writing `Library` on top is just annoying.

Reply:
Fair enough. lgtm.